src/Query.jl

```diff
@@ -251,15 +264,11 @@ end
 
 "Execute a query, flattening blocks into a dataframe."
 function select_df(
-    sock::ClickHouseSock,
-    query::AbstractString;
+    query::AbstractString,
+    sock::ClickHouseSock = connect();
     kwargs...
 )::DataFrame
-    columns = pairs(select(sock, query; kwargs...))
-    DataFrame(
-        [x for (_, x) ∈ columns],
-        [x for (x, _) ∈ columns],
-    )
+    DataFrame(select(query, sock; kwargs...))
```

Comment:
That's pretty neat -- I didn't know that `DataFrame` had this constructor -- nice!